plugins/basic/nexus-repository-raw/src/main/java/org/sonatype/nexus/repository/raw/internal/RawGroupRecipe.groovy

```diff
@@ -89,7 +94,14 @@ class RawGroupRecipe
     Router.Builder builder = new Router.Builder()
 
     builder.route(new Route.Builder()
-        .matcher(new TokenMatcher("/{name:.+}"))
+        .matcher(new SuffixMatcher('/', false))
```

Comment:
Ah, I see now why you have it as a separate route. I had imagined this feature as a handler that sat near the top of the stack, and when it got a 404 for .endsWith("/"), it would re-issue another request for path + suffix.